include/uv-win.h

```diff
@@ -609,6 +609,7 @@ RB_HEAD(uv_timer_tree_s, uv_timer_s);
       uv_buf_t bufsml[4];                                                     \
     } info;                                                                   \
     struct {                                                                  \
+      double btime;                                                           \
```

Comment:
Yeah, @bnoordhuis already commented on this.  Looks like this won't land in `v1.x`.